mm2src/coins/z_coin.rs

```diff
@@ -2163,6 +2163,79 @@ impl InitWithdrawCoin for ZCoin {
     }
 }
 
+impl ZCoin {
+    #[cfg(feature = "run-docker-tests")]
+    pub async fn withdraw_for_tests(&self, req: WithdrawRequest) -> Result<TransactionDetails, MmError<WithdrawError>> {
+        if req.fee.is_some() {
+            return MmError::err(WithdrawError::UnsupportedError(
```

Comment:
i think the reason for not using the other `init_withdraw/status_withdraw` is that it's task managed and need an rpc and a `MarketMakerIt` to start?

can't we call `init_withdraw` here and provide a dummy `task_handle` just for processing this task and polling the status?

Reply:
Yes, I'd like to.
I remember in the past I once tried to create a dummy task handle but that would have required too much extra code and some task manager refactoring. I was not sure it was worth it then and now I did not go that way too.